platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/PkiTools.java

```diff
@@ -243,13 +268,12 @@ public static X500Name makeDistinguishedName(String commonName) {
     public static X509Certificate pemToCertificate(String certString) {
         CertificateFactory cf = new CertificateFactory();
         ByteArrayInputStream in = new ByteArrayInputStream(PkiTools.pemToDer(certString));
-        X509Certificate cert = null;
+        X509Certificate cert;
```

Comment:
How do you stop IntelliJ (findbugs?) from complaining about possible uninitialized value for ` X509Certificate cert;`. I don't like redundantly setting values like that `null`, but I don't know an easy way to suppress the quality tools.